pkg/xmtp/listener.go

```diff
@@ -139,11 +138,6 @@ func (l *Listener) startMessageWorkers() {
 }
 
 func (l *Listener) processEnvelope(env *v1.Envelope) error {
-	if shouldIgnoreTopic(env.ContentTopic) {
```

Comment:
Should we replace `shouldIgnoreTopic` with something like `if !isV3Topic(env.ContentTopic)`.

The more messages we can filter out before we hit the database on line 147, the better.